README.md

```diff
@@ -29,6 +29,10 @@
 > - We've released v0.5.0-beta. You can find release notes [here](https://github.com/linkedin/datahub/releases/tag/v0.5.0-beta)
 > - We're on [Slack](docs/slack.md) now! Ask questions and keep up with the latest announcements.
 
+✨ DataHub participates in Hacktoberfest. Refer https://hacktoberfest.digitalocean.com for more details.
```

Comment:
Move this to top. Also indent the first line like the others

Reply:
Done